app/grandchallenge/algorithms/templates/algorithms/job_list_row.html

```diff
@@ -11,13 +11,13 @@
    title="View result details">
     <i class="fa fa-info-circle"></i>
 </a>
-<split/>
+<split></split>
```

Comment:
I think we need to revert these changes and all similar ones in the other templates. Or we need to change the processing here https://github.com/comic/grand-challenge.org/blob/master/app/grandchallenge/datatables/views.py#L30, which may be a better option.

Reply:
The change to `datatables/views.py` is included in this PR.

Reply:
Wow, sorry, I somehow completely missed that.

Reply:
That's ok! The line is also covered well by tests.